src/components/remotecontrol/remotecontrol.js

```diff
@@ -650,12 +708,40 @@ define(['browser', 'datetime', 'backdrop', 'libraryBrowser', 'listView', 'imageL
                     playbackManager.fastForward(currentPlayer);
                 }
             });
-            context.querySelector('.btnPreviousTrack').addEventListener('click', function () {
+            for (let shuffleButton of context.querySelectorAll('.btnShuffleQueue')) {
+                shuffleButton.addEventListener('click', function () {
+                    if (currentPlayer) {
+                        if (playbackManager.getQueueShuffleMode(currentPlayer) === 'Sorted') {
+                            playbackManager.setQueueShuffleMode('Shuffle', currentPlayer);
+                        } else {
+                            playbackManager.setQueueShuffleMode('Sorted', currentPlayer);
+                        }
+                    }
+                });
+            }
+
+            context.querySelector('.btnPreviousTrack').addEventListener('click', function (e) {
+                if (currentPlayer) {
+                    if (currentPlayer.id === 'htmlaudioplayer' && (currentPlayer._currentTime >= 5 || !playbackManager.previousTrack(currentPlayer))) {
```

Comment:
```suggestion
                    if (currentPlayer.id === 'htmlaudioplayer' && (currentPlayer._currentTime >== 5 || !playbackManager.previousTrack(currentPlayer))) {
```

Reply:
@MrTimscampi This gives ``unexpected token error``